seqr/views/apis/variant_search_api_tests.py

```diff
@@ -96,7 +96,6 @@
     'rnaSeqData': {'I000001_na19675': {'outliers': {'ENSG00000268903': mock.ANY}}},
     'phenotypeGeneScores': {
         'I000001_na19675': {'ENSG00000268903': {'exomiser': EXPECTED_EXOMISER_DATA}},
-        'I000002_na19678': {'ENSG00000268903': {'lirical': EXPECTED_LIRICAL_DATA}},
```

Comment:
Can we change the rank on the other entry fixture, or add a new fixture with lower rank? I do still want to test lirical data in search results

Reply:
Sure